examples/kg-chat/README.md

```diff
@@ -0,0 +1,71 @@
+# PyPi Packages Dependency Chatbot
+
+This example uses a `DependencyGraphAgent` 
+(derived from [`Neo4jChatAgent`](https://github.com/langroid/langroid/blob/main/langroid/agent/special/neo4j/neo4j_chat_agent.py)).
+It auto-generates a `neo4j` knowledge-graph based on the dependency
+structure of a given `PyPi` package. You can then ask the chatbot questions
+about the dependency graph. This agent uses two tools in addition to those 
+already available to `Neo4jChatAgent`:
+
+- GoogleSearchTool to find package version and type information 
```

Comment:
any questions actually can be forwarded to this tool, like after obtaining the package and version in the dependency graph, you can search the web about if this version is vulnerable or not.

Reply:
> you can search the web about if this version is vulnerable or not.

ok please update the readme to say that